chainer/functions/array/vstack.py

```diff
@@ -49,11 +49,32 @@ def vstack(xs):
     """Concatenate variables vertically (row wise).
 
     Args:
-        xs (list of chainer.Variable): Variables to be concatenated.
+        xs (list of :class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variables to be concatenated. The variables must have the
+            same shape, except in the vertical (row wise) dimension.
```

Comment:
Please describe the behavior when input arrays does not have horizontal dimension (i.e. ndim == 0).